pkg/config/constants.go

```diff
@@ -76,9 +76,9 @@ const (
 	// SparkNodeSelectorKeyPrefix is the configuration property prefix for specifying node selector for the pods.
 	SparkNodeSelectorKeyPrefix = "spark.kubernetes.node.selector."
 	// SparkDriverContainerImageKey is the configuration property for specifying a custom driver container image.
-	SparkDriverContainerImageKey = "spark.kubernetes.driver.container.image"
+	SparkDriverContainerImageKey = "spark.kubernetes.driver.docker.image"
```

Comment:
why this change?

Reply:
driver/executor image config keys were modified from 2.2 to 2.3. This is in-sync with 
https://github.com/apache-spark-on-k8s/spark/blob/branch-2.2-kubernetes/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/config.scala#L38

Reply:
ok, cool